Rubberduck.Core/UI/CustomComWrappers/IOleWrapper.cs

```diff
@@ -298,14 +329,18 @@ public Wrapper_IOleInPlaceFrame(IntPtr hostObjectPtr) : base(hostObjectPtr)
             }
         }
 
-        public override void Dispose()
+        protected override void Dispose(bool disposing)
```

Comment:
Ok, this was funky but also, thar be big scary dragons here. Please test the changes with a native debugger attached and ensure no exceptions get thrown on exit. This is a part of the code responsible for preventing exceptions / AVs that happens long after the AppDomain shutdown.

Reply:
I'm getting a clean exit here with Excel.